src/satipc.c

```diff
@@ -1040,6 +1040,8 @@ void satipc_commit(adapter *ad)
 	if (sip->lap + sip->ldp == 0)
 		if (!sip->force_commit || !ad->tp.freq)
 			return;
+	if (sip->ldp > 0 && sip->last_connect == 0 && !sip->want_tune)
```

Comment:
This case appears if the session is already open (transponder) but a new adapter is created. In this case, the DELETE PIDS is commited. But in this case, nothing requires to be done. This check resolves the problem.

Reply:
Hi @catalinii ,

Please, don't commit this at time! Perhaps this line is wrong.

Reply:
Now fixed!